repositories.bzl

```diff
@@ -55,28 +163,12 @@ def ign_math_repositories():
     eigen3()
 
 def ign_common_repositories():
-    native.new_local_repository(
-        name = "glib",
-        path = "/usr/include/glib-2.0",
-        build_file_content = """
-package(default_visibility = ["//visibility:public"])
-cc_library(
-    name = "headers",
-    hdrs = glob(["**/*.h"])
-)
-""",
-    )
-    native.new_local_repository(
-        name = "glibconfig",
-        path = "/usr/lib/x86_64-linux-gnu/glib-2.0/include",
-        build_file_content = """
-package(default_visibility = ["//visibility:public"])
-cc_library(
-    name = "headers",
-    hdrs = glob(["**/*.h"])
-)
-""",
-    )
+    dl()
```

Comment:
Think this needs to be deleted; I noticed you deleted the corresponding function

Reply:
+1

Reply:
Removed in latest.